src/pages/index.js

```diff
@@ -181,7 +178,7 @@ Promise.all([api.getUserInformation(), api.getInitialCards()])
     userInfo.setUserInfo(userData);
     userInfo.setUserProfileImage(userData);
     initialCards.forEach((card) => addCard(card));
```

Comment:
Please, delete line 180. You have `renderItems` for rendering the array of cards. Please, use it